factory/random.py

```diff
@@ -22,3 +24,6 @@ def reseed_random(seed):
     """Reseed factory.fuzzy's random generator."""
     r = random.Random(seed)
     set_random_state(r.getstate())
+
+    for locale in Faker._FAKER_REGISTRY:
+        Faker._FAKER_REGISTRY[locale].random.setstate(r.getstate())
```

Comment:
I think it makes more sense to keep this code in `factory.random` morethan `factory.faker`. No?